design-system/src/_css/main.css

```diff
@@ -14,18 +14,19 @@
 @import "@coopdigital/foundations-tables";
 @import "@coopdigital/foundations-layout";
 
+/* TODO: Fix components with typos in published path */
```

Comment:
Sadly a couple of packages had typos in their paths, we can revert this after the next `npm publish`